src/resources/views/crud/buttons/delete.blade.php

```diff
@@ -14,29 +14,14 @@
 	  function deleteEntry(button) {
 		// ask for confirmation before deleting an item
 		// e.preventDefault();
-		var button = $(button);
-		var route = button.attr('data-route');
-		var row = $("#crudTable a[data-route='"+route+"']").closest('tr');
+		var route = $(button).attr('data-route');
 
 		swal({
 		  title: "{!! trans('backpack::base.warning') !!}",
 		  text: "{!! trans('backpack::crud.delete_confirm') !!}",
 		  icon: "warning",
-		  buttons: {
-		  	cancel: {
-			  text: "{!! trans('backpack::crud.cancel') !!}",
-			  value: null,
-			  visible: true,
-			  className: "bg-secondary",
-			  closeModal: true,
-			},
-		  	delete: {
-			  text: "{!! trans('backpack::crud.delete') !!}",
-			  value: true,
-			  visible: true,
-			  className: "bg-danger",
-			}
-		  },
+		  buttons: ["{!! trans('backpack::crud.cancel') !!}", "{!! trans('backpack::crud.delete') !!}"],
+		  dangerMode: true,
```

Comment:
I agree with the change since is a sensible operation. 

But wouldn't this become a BC now ? Atleast from UI/UX perspective. If people are used to click delete and "Enter", now the confirmation will be closed as `Cancel` is the focused button, while previously was the `Delete`.

Reply:
Personally I wouldn’t consider this a BC, if someone is using the mouse to control backpack it has zero impact (which I assume most users do?). Only in your example mouse/keyboard cancel is the new default.
- From a data safety standpoint, cancel is the better option to prevent accidental deletions.
- If someone is using the mouse keyboard combo, they will recognize cancel is the new default after they tried once

Reply:
Indeed @martijnb92 we agree on the importance of beeing safe in this scenario and use `Cancel` as the default focus option. 

I am still not sure about not beeing a BC, maybe not a Wreck style BC, but a small behaviour change. 
If we all agree on that I wouldn't find that change a big of an issue, but I am not everyone, so we better be aware what will happen if we do it :)

Let's hear from @tabacitu and @promatik here. 

Thanks again @martijnb92 👍

Reply:
Good catch @pxpm, I tend to use keyboard whenever I can, I'll need to do delete > **tab** > enter now 😅

Anyway this is for the better. I agree with the change too 👍